### PR TITLE
Update benchmark and control tags for compatibility with dashboard page Closes #26

### LINKED ISSUE #26: Update benchmark and control tags for compatibility with dashboard page
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by khushboo9024**
### Checklist
- [ ] Issue(s) linked
image

image

image
image

### CODE REVIEW

File: controls/sql.sp [hunk 1]
```diff
@@ -11,19 +11,22 @@ variable "sql_database_age_warning_days" {
 }
 
 locals {
-  sql_common_tags = merge(local.thrifty_common_tags, {
-    service = "sql"
+  sql_common_tags = merge(local.azure_thrifty_common_tags, {
+    service = "Azure/SQLDatabase"
```
Comment (cbruno10) [inline controls/sql.sp]:
```suggestion
    service = "Azure/SQL"
```

Comment (cbruno10) [reply controls/sql.sp]: If unsure of what the service should be called, please check the Azure Insights mod first and see what it's called there.

**Review by cbruno10 — CHANGES_REQUESTED** [review]
Please see suggestion, thanks!

**Review by rajlearner17 — APPROVED** [review]
LGTM

**Pull request merged**